kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/DocumentationUrls.java

```diff
@@ -65,7 +65,7 @@ public enum DocumentationUrls {
      */
     JFXCENTRAL_HOMEPAGE("https://www.jfx-central.com");
 
-    private static final String SAFE_DEFAULT_VERSION = "21";
+    private static final String SAFE_DEFAULT_VERSION = "22";
```

Comment:
Why is this set to 22?

Reply:
My bad, I changed it thinking it was the JDK version... I've reverted it now.